blank-canvas-blocks/experimental-theme.json

```diff
@@ -161,6 +175,15 @@
 					"thickness": "2px",
 					"width": "150px"
 				},
+				"search": {
+					"button": {
+						"lineHeight": "27px",
+						"padding": "calc( .9 * var(--wp--custom--padding--horizontal) )"
+					},
+					"input": {
+						"margin": "0 calc( .66 * var(--wp--custom--padding--horizontal)) 0 0"
```

Comment:
In Gutenberg, the search button styles have a `margin-left` applied rather than the input. 

<img width="426" alt="Screen Shot 2021-03-16 at 12 31 45 PM" src="https://user-images.githubusercontent.com/5375500/111345606-b2261f00-863a-11eb-9470-604101dd2fbd.png">

Do you think it would make sense to add margin to the button (overwriting Gutenberg's styles) instead of adding a new margin control to the input element? Right now, the result is two margins sitting next to each other:

<img width="788" alt="Screen Shot 2021-03-16 at 12 34 34 PM" src="https://user-images.githubusercontent.com/5375500/111345868-e4378100-863a-11eb-94c6-8df89febd447.png">